src/Testcontainers.LocalStack/LocalStackContainer.cs

```diff
@@ -0,0 +1,41 @@
+namespace Testcontainers.Minio;
+
+/// <inheritdoc cref="DockerContainer" />
+[PublicAPI]
+public sealed class LocalStackContainer : DockerContainer
+{
+    private readonly LocalStackConfiguration _configuration;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="LocalStackContainer" /> class.
+    /// </summary>
+    /// <param name="configuration">The container configuration.</param>
+    /// <param name="logger">The logger.</param>
+    public LocalStackContainer(LocalStackConfiguration configuration, ILogger logger)
+        : base(configuration, logger)
+    {
+        _configuration = configuration;
+    }
+
+    /// <summary>
+    /// Gets the AWS access key id.
+    /// </summary>
+    /// <returns>The AWS access key id.</returns>
+    public string GetAccessKeyId() => "dummy";
+
+    /// <summary>
+    /// Gets the AWS access secret.
+    /// </summary>
+    /// <returns>The AWS access secret.</returns>
+    public string GetAccessSecret() => "dummy";
```

Comment:
We do not really need `GetAccessKeyId()` and `GetAccessSecret()` right? What do you think if we use something like this in `LocalStackContainerTest`:

https://github.com/testcontainers/testcontainers-dotnet/blob/9f86d9f273e460b909c76df35177259fa674fd42/tests/Testcontainers.DynamoDb.Tests/DynamoDbContainerTest.cs#L7-L11

Reply:
Your solution is much nicer, I fixed the code